tree/dataframe/src/RForestDS.cxx

```diff
@@ -67,7 +67,7 @@ RDF::RDataSource::Record_t RForestDS::GetColumnReadersImpl(std::string_view name
 }
 
 bool RForestDS::SetEntry(unsigned int /*slot*/, ULong64_t entryIndex) {
-   fForest->SetEntry(entryIndex, fEntry.get());
+   fForest->LoadEntry(entryIndex, fEntry.get());
```

Comment:
Does the I/O operation happens during that function (as the name indicates) or is it delayed until use?

Reply:
`LoadEntry()` is synchronous and the I/O work happens during the call.

Reply:
fair enough, then it is a good name.